config/libc/glibc.in

```diff
@@ -416,4 +417,10 @@ config GLIBC_ENABLE_WERROR
 
 endif
 
+config GLIBC_ENABLE_COMMON_FLAG
+    bool "Enable -fcommon flag for older version of glibc when using GCC >=10"
+    default y if GLIBC_2_29_or_older && GCC_10_or_later
+    default n if GLIBC_2_30_or_later || GCC_9_or_older
+
```

Comment:
Since we're here anyway may as well remove one of these blank lines

Reply:
I removed extra empty lines and added help text.